src/dos/dos_locale.cpp

```diff
@@ -2576,6 +190,33 @@ static void maybe_log_changed_country(const std::string &country_name,
 	logged_config_period = config.locale_period;
 }
 
+static LocaleSeparator get_default_list_separator(const LocaleSeparator thousands_separator,
+                                                  const LocaleSeparator decimal_separator)
+{
+	// Current locale systems (like Unicode CLDR) do not seem to specify
+        // list separators anymore.
+	// On 'https://answers.microsoft.com' one can find a question:
+	// "Why does Excel seem to use ; and , differently per localization?"
+	// And the top answer is:
+	// "In countries such as the USA and UK, the comma is used as list
+        // separator. (...) In countries that use comma as decimal separator,
+        // such as many continental European countries, using a comma both as
+	// decimal separator and as list separator would be very
+	// confusing: does 3,5 mean the numbers 3 and 5, or does it mean
+	// 3 and 5/10? So, in such countries, the semi-colon ; is use
+	// as a list separator".
+	// - https://answers.microsoft.com/en-us/msoffice/forum/all/why-does-excel-seem-to-use-and-differently-per/6467032f-43a0-4343-bae7-af8853fec754
+	//
+	// Let's use this algorithm to determine default list separator.
+
+	if (thousands_separator != LocaleSeparator::Comma &&
+	    decimal_separator != LocaleSeparator::Comma) {
+		return LocaleSeparator::Comma;
+	} else {
+		return LocaleSeparator::Semicolon;
+	}
+}
```

Comment:
I don't really get this. You use this in `refresh_time_date_format()` only, apparently. But aren't you emulating known existing DOS locales, so we should know what separator was used in DOS (or is used in FreeDOS)?

Then if we don't do this for emulating anything DOS specific, I'd just use semicolons for all locales (hardcode it), and that's the end of it.

Reply:
OK, I am now using semicolons if there is no definition for the given country.